$OpenBSD: patch-c_stuff_lib_fb_stuff_pm,v 1.1 2009/06/18 12:54:28 martynas Exp $

Don't preserve premissions from shared data files, since we want
local files to be writable.

--- c_stuff/lib/fb_stuff.pm.orig	Sat Dec  6 16:12:39 2008
+++ c_stuff/lib/fb_stuff.pm	Wed Jun 17 21:46:19 2009
@@ -244,7 +244,6 @@ sub cp_af {
 	    local *F; open F, $src or die "can't open $src for reading: $!\n";
 	    local *G; open G, "> $dest";
 	    local $_; while (<F>) { print G $_ }
-	    chmod((stat($src))[2], $dest);
 	}
     }
     1;
